save-cloud-common/src/commonMain/kotlin/com/saveourtool/save/entities/benchmarks/BenchmarkEntity.kt

```diff
@@ -19,6 +79,23 @@ enum class BenchmarkCategoryEnum {
     PERFORMANCE,
     STATIC_ANALYSIS,
     ;
+
+    companion object : MenuBar<BenchmarkCategoryEnum> {
+        override val defaultTab = ALL
+        val listOfStringEnumElements = BenchmarkCategoryEnum.values().map { it.name.lowercase() }
+        override val regex = Regex("/project/[^/]+/[^/]+/[^/]+")
+        override var paths: Pair<String, String> = "" to ""
```

Comment:
Why do you always override paths?

Reply:
Maybe it's better to remove paths from the interface?

Reply:
Probably

Reply:
Or at least make it to be separate variables instead of pair with name that gives nothing about it's usage